payment-instruments/card/features/optional-features/payout.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Payout
+redirect_from:
+estimated_read: 3
+description: |
+  Welcome to Payout.
+  This section has extented code examples and features that were not
+  covered by the other subsections.
```

Comment:
Repeating this in every description isn't useful. Please describe the feature, here being "Payout", instead.